services/storage/tests/test_python_client_sdk.py

```diff
@@ -0,0 +1,38 @@
+# pylint:disable=unused-variable
+# pylint:disable=unused-argument
+# pylint:disable=redefined-outer-name
+
+from typing import Dict
+from pathlib import Path
+import pytest
+
+HERE = Path(__file__).resolve().parent
```

Comment:
FYI: there are more reliable fixture to paths in a project in ``packages/pytest-simcore/src/pytest_simcore/repository_paths.py`` . 

In any case, I noticed that ``services/storage/tests/conftest.py`` is not very up-to-date as e.g. ``services/catalog/tests/unit/conftest.py``.

Reply:
Thanks, good one here, I've used the fixture in the end.